cropduster/tests/settings.py

```diff
@@ -31,3 +40,21 @@
     AWS_S3_SIGNATURE_VERSION = 's3v4'
 else:
     DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
+
+CKEDITOR_CONFIGS = {
+    'default': {
+        'extraPlugins': 'cropduster',
+        'removePlugins': 'flash,forms,contextmenu,liststyle,table,tabletools,iframe',
+        'disableAutoInline': True,
+        "height": 450,
+        "width": 840,
+        'cropduster_uploadTo': 'ckeditor',
+        'cropduster_previewSize': [570, 300],
+        'cropduster_url': lazy_reverse('cropduster-standalone'),
+        'cropduster_urlParams': {'max_w': 672, 'full_w': 960},
+    },
+}
+
+CKEDITOR_UPLOAD_PATH = "%s/upload" % MEDIA_ROOT
+
+os.makedirs(CKEDITOR_UPLOAD_PATH)
```

Comment:
Is this supposed to be file based? I thought we had done work elsewhere in cropduster to make the storage backend not tied to the file system.

Reply:
This is a dumb requirement of django-ckeditor that should be fixed. It's a django-filebrowser integration thing (which is incompatible with S3, as you know). Importing django-ckeditor throws a `ImproperlyConfigured` exception if this setting is unset or it doesn't exist on the local filesystem, regardless of whether you're using the django-filebrowser feature.